homeassistant/components/lifx/migration.py

```diff
@@ -0,0 +1,74 @@
+"""Migrate lifx devices to their own config entry."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr, entity_registry as er
+
+from .const import _LOGGER, DOMAIN
+from .discovery import async_init_discovery_flow
+
+
+async def async_migrate_legacy_entries(
```

Comment:
These functions don't need to be coroutine functions since we don't await inside.

Reply:
https://github.com/home-assistant/core/pull/75719

👍